pv_site_api/_db_helpers.py

```diff
@@ -309,6 +309,8 @@ def check_user_has_access_to_site(session: Session, auth: dict, site_uuid: str):
             f"User has access to {site_uuids}",
         )
 
+    return user
```

Comment:
This is a code smell. This function should only ever return a boolean – which IMO it should return, and the exception can be thrown in the caller side, so that we can use this function internally in a more explicit way... but I'll settle for putting it back to how it was for now